### ISSUE TITLE
using with windows executor

### ISSUE THREAD

**Issue by nicks** [CircleCI-Public/slack-orb#145]
### Orb version
3.4.0

### What happened

On all my builds, I have

```
      - slack/notify-on-failure:
          only_for_branches: master
```

I'd like to also add this for my Windows executor.

```

  build-windows:
    executor:
      name: win/default
      size: "medium"
    steps:
      - checkout
      - run: go test ./...
      - slack/notify-on-failure:
          only_for_branches: master
```

This fails with the following error:

```
Provide error if non-bash shell
#!powershell.exe -ExecutionPolicy Bypass
if [ ! -x /bin/bash ]; then
  echo Bash not installed.
  exit 1
fi
```

I understand why the error is happening. But I'm not sure if there's an alternative way to accomplish this. Any ideas?

**Comment by sadabailey** [comment]
@sadabailey

**Comment by sadabailey** [comment]
@sadabailey

**Comment by chytreg** [comment]
Willing to see solution for windows/executor as well. There is `bash.exe` which we can use and hopefully most of the code would work.
On windows/executor I use smt like this for bash scripts:
```
      - run:
          name: Some name
          command: |
            touch fromci.txt
            ./bin/script.sh ./fromci.txt
          shell: bash.exe
```

**Comment by chytreg** [comment]
@nicks `shell: bash.exe` and https://github.com/coursehero/slacktee works for me as a workaround

**Comment by sadabailey** [comment]
I'm really sure out of my scripts thanks I'll try it

**Comment by mickmcbride** [comment]
> @nicks `shell: bash.exe` and https://github.com/coursehero/slacktee works for me as a workaround

Do you have an example of this working? 

I'm running into the same issues with the windows executor and the slack orb.

**Comment by KyleTryon** [comment]
Hello all, just wanted to let everyone know we will be looking into adding Windows support for the slack orb in the next major update coming soon: https://github.com/CircleCI-Public/slack-orb/tree/Major-block-kit-redesign

**Comment by EricRibeiro** [comment]
Hey, @nicks 👋 

Support for Windows was implemented in [v4.12.0](https://github.com/CircleCI-Public/slack-orb/releases/tag/v4.12.0). If you want to give it a shot, make sure to choose `bash.exe` as your [shell in the Windows executor](https://circleci.com/docs/using-windows/#specifying-a-shell-with-the-windows-executor).

Let me know how it goes!
